spec/spec_helper.rb

```diff
@@ -23,8 +23,8 @@
     add_filter 'lib/feature_flipper.rb'
     add_filter 'spec/support/authenticated_session_helper'
     add_filter 'spec/support/attr_encrypted_matcher'
-    add_filter 'app/models/health_beta_registration.rb'
-    add_filter 'app/controllers/v0/health_beta_registrations_controller.rb'
+    add_filter 'app/models/beta_registration.rb'
+    add_filter 'app/controllers/v0/beta_registrations_controller.rb'
```

Comment:
If the plan is to leave this in as a generic model / controller, do we maybe want to add specs around these rather than skipping them in coverage?